package.json

```diff
@@ -33,6 +33,10 @@
     },
     "main": "./out/src/extension",
     "contributes": {
+        "commands": [
+            { "title": "w", "command": "workbench.action.files.save" },
+            { "title": "q", "command": "workbench.action.closeActiveEditor" }
+        ],

```

Comment:
I think we should implement our own class to handle vim command using `showInputBox`.
Register commands to vscode's command palette will confuse normal usage of it.